integrationtests/k8s_test.go

```diff
@@ -69,6 +69,7 @@ func TestK8s(t *testing.T) {
 		ok := TestConnection(masterIP, "22")
 		assert.True(t, ok)
 
+		time.Sleep(10 * time.Second)
```

Comment:
i remember in our discussion that the test sometimes failed because the workers were not yet ready. so wouldn't retrying for some duration be better that to wait for some time once then try??